src/notifications/lipsticknotification.h

```diff
@@ -46,6 +46,7 @@ class LIPSTICK_EXPORT LipstickNotification : public QObject
     Q_PROPERTY(int priority READ priority NOTIFY priorityChanged)
     Q_PROPERTY(QString category READ category NOTIFY categoryChanged)
     Q_PROPERTY(bool userRemovable READ isUserRemovable NOTIFY userRemovableChanged)
+    Q_PROPERTY(QVariantList remoteActions READ remoteActions)
```

Comment:
Shouldn't the property be marken CONSTANT then? That way there won't be warnings when some qml code binds a property to it.